metrics/web/lib/project_groups/presentation/view_models/edit_project_group_dialog_view_model.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:metrics/project_groups/presentation/view_models/project_group_dialog_view_model.dart';
+
+/// A class that represents the data of the project group to display
+/// within dialogs for adding and updating a project group.
+class EditProjectGroupDialogViewModel extends ProjectGroupDialogViewModel {
```

Comment:
What do you think if we don't extend anything here and copy the required fields like we do in ProjectGroupCardViewModel? It feels like we could change the naming then to just ProjectGroupDialogViewModel and then in other parts naming of the methods would be improved as well. See here:
![image](https://user-images.githubusercontent.com/442086/86282551-21acc900-bbe8-11ea-8c35-b85a9007e289.png)